main.js

```diff
@@ -0,0 +1,12 @@
+var day = document.querySelector('.week');
+var schedule = document.querySelector('.event-viewer');
+
+day.addEventListener('click', dayOfWeek);
+
+function dayOfWeek() {
+  var sentence = event.target.textContent;
+  if (event.target.className === 'week') {
```

Comment:
it would probably make sense to check if the element clicked is the square with the day in it
you can solve this by adding a class of day on the element and doing:
`if (event.target.className !== 'day') return;`
or you can just make sure it is an h2, I like the first way better but this works too
`if(event.target.tagName !== 'H2') return;`